### PR TITLE
write app-state.json again so installer understands gui busy state

### PR DESCRIPTION
**Pull request by chrisnojima**
This fixes the busy check here:
https://github.com/keybase/go-updater/blob/0998111b6e4a0d9d1cd8e797b669e158c5d3ddf0/updater.go#L317

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/desktop/app/node.desktop.tsx [hunk 1]
```diff
@@ -213,6 +221,19 @@ const findRemoteComponent = (windowComponent: string, windowParam: string) => {
 const plumbEvents = () => {
   Electron.app.on('KBkeybase' as any, (_: string, action: Action) => {
     switch (action.type) {
+      case 'activeChanged':
+        // the installer reads this file to understand the gui state to not interrupt
+        // TODO change how this works
+        try {
+          fs.writeFileSync(
```
Comment (buoyad) [inline shared/desktop/app/node.desktop.tsx]: Would this remove the other data in this file? Mine has the window location/size info there.

Comment (chrisnojima) [reply shared/desktop/app/node.desktop.tsx]: yeah, but thats historical and ignored. thats since moved to gui_config.json and owned by the daemon

**Pull request merged**